web_app_drawer/static/src/less/dropdowns.less

```diff
@@ -0,0 +1,55 @@
+/* Copyright 2016 LasLabs Inc.
+ * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
+
+.dropdown-submenu {
```

Comment:
Hmmmm it seems they have a scroll on their dropdowns. IIRC we don't have that- pretty sure these things will go past the screen. Will double check though & see if I can find some dropdowns that aren't bootstrap but already exist in Odoo 9c